examples/tutorials/core_api_pytorch_mnist/model_def_metrics.py

```diff
@@ -69,7 +69,7 @@ def train(args, model, device, train_loader, optimizer, epoch, core_context):
             # NEW: Report epoch-based training metrics to Determined master via core_context.
             # Index by (batch_idx + 1) * (epoch-1) * len(train_loader) to continuously plot loss on one graph for consecutive epochs.
             core_context.train.report_training_metrics(
-                steps_completed=(batch_idx + 1) + (epoch - 1) * len(train_loader),
+                steps_completed=(batch_idx + 1) + epoch * len(train_loader),
                 metrics={"train_loss": loss.item(), "epoch": epoch},
```

Comment:
because I changed epoch indexing to start from 0 rather than 1 in the train loop, I also changed this part to make sure the metric value did not change